### ISSUE TITLE
A trained model for Music Genre Classification

### ISSUE THREAD

**Issue by vybhav72954** [HarshCasper/Rotten-Scripts#576]
**Is your feature request related to a problem? Please describe.**
A pre-trained model for music genre classification is not tough to find. A good model for music genre classification is like a needle in haystack.

**Describe the solution you'd like**
After working on Music classification for more than a year, I believe I can come up with a really good model. I will be using  Benchmark dataset, `GTZAN`, `Emotify` for example, Python offers TF, Keras and most importantly  `pyaudioanalysis` so i think Python is the best choice.

**Describe alternatives you've considered**
There are multiple models available, but either they are now old, they are using old algos like `SVM`, or old variants of Neighbours. 

**Additional context**

**Comment by vybhav72954** [comment]
@HarshCasper I assigned it myself cause I don't want others to work on it before it is assigned to me. It's a big task and if someone would open a PR before me, it would be wrong to close it as the script is not easy. Can you put the NJACK label so that I can start?

**Comment by HarshCasper** [comment]
Hello @vybhav72954

Are you still working on the same?

**Comment by vybhav72954** [comment]
Tomorrow Night at max, if I won't be able to solve this, I will unassing my self for sure

**Comment by vybhav72954** [comment]
@HarshCasper So I was just going to make a PR,  but just one doubt, I don't want to upload a heavy well-trained model for the sake of the size of Repo, all I want to do is upload the most basic implementation of Deep learning for Music classification. I can explain steps in README that will give an idea to make it more efficient, but am I allowed to link my own REPOSITORY in the README, if people are interested in a more complex implementation, like combining classifiers, multi-depth classification, ML approach, AI approach and the theory involved.

I am opening a DRAFT pr as of now, kindly solve my dilemma and  I will do the needful.

P.S. - Don't worry about the code I will upload, it's not that bad XD. It is actually far better than my first ever successful implementation and is one of the simpler version of a script that i am trying to get Published.

**Comment by HarshCasper** [comment]
> @HarshCasper So I was just going to make a PR, but just one doubt, I don't want to upload a heavy well-trained model for the sake of the size of Repo, all I want to do is upload the most basic implementation of Deep learning for Music classification. I can explain steps in README that will give an idea to make it more efficient, but am I allowed to link my own REPOSITORY in the README, if people are interested in a more complex implementation, like combining classifiers, multi-depth classification, ML approach, AI approach and the theory involved, my Repo has it all
> 
> I am opening a DRAFT pr as of now, kindly solve my dilemma and I will do the needful.
> 
> P.S. - Don't worry about the code I will upload, it's not that bad XD. It is actually far better than my first ever successful implementation and is one of the simpler version of a script that i am trying to get Published.

Yeah. Please go ahead! 

Regarding the Model Size, you can upload it on a Google Drive and reference it there just for sake of it.
